firefox_ui_tests/remote/security/test_safe_browsing_notification.py

```diff
@@ -56,11 +62,16 @@ def test_notification_bar(self):
 
                 # Navigate to the unsafe page
                 # Check "ignore warning" link then notification bar's "not badware" button
-                self.marionette.navigate(unsafe_page)
-                # Wait for the DOM to receive events for about:blocked
-                time.sleep(1)
-                self.check_ignore_warning_button(unsafe_page)
-                self.check_not_badware_button(label, report_page)
+                # Only do this if feature supports it
+                if label is not None:
+                    self.marionette.navigate(unsafe_page)
+                    # Wait for the DOM to receive events for about:blocked
+                    time.sleep(1)
+                    self.check_ignore_warning_button(unsafe_page)
+                    self.check_not_badware_button(label, report_page)
+                else:
+                    # Must reset permissions if this test did not run previously
+                    self.utils.remove_perms('www.itisatrap.org', 'safe-browsing')
```

Comment:
Removing the permissions should only be necessary when the ignore warning link gets clicked. And therefore we already have the removal of permissions in `check_ignore_warning_button`. So I don't see why this is necessary here.